### PR TITLE
Add atomic-guest tuned profile

### PR DESCRIPTION
**Pull request by andrewklau**

### CODE REVIEW

**Comment by detiber** [comment]
ok to test

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
lgtm

**Comment by sdodson** [comment]
Other than being surprised that this isn't already the default for atomic host, LGTM.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

File: roles/openshift_node/tasks/main.yml [hunk 1]
```diff
@@ -34,6 +34,10 @@
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }},tuned-profiles-{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Set atomic-guest tuned profile
+  action: "tuned-adm profile atomic-guest"
```
Comment (sdodson) [inline roles/openshift_node/tasks/main.yml]: command rather than action.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by andrewklau** [comment]
hm any ideas why it error'ed?

**Comment by sdodson** [comment]
It errored out when running the e2e tests post installation. But only for 3.2 so I'm assuming it's a flake. We'll know for sure if all future CI runs fail on 3.2 containerized installs but I think that's not happening.

**Comment by sdodson** [comment]
aos-ci-test

**Pull request merged**